src/modules/taskList.js

```diff
@@ -0,0 +1,126 @@
+import Task from './task.js';
+import Status from './status.js';
+
+export default class TaskList {
+  constructor() {
+    this.listArray = [];
+  }
+
+  display() {
+    const listContainer = document.querySelector('.listContainer');
+    if (listContainer) {
+      listContainer.innerHTML = '';
+    }
+    this.listArray.forEach((el) => {
+      const li = [];
+      const input = [];
+      const p = [];
+      const button = [];
+      // Create List Element
+      li[el.index] = document.createElement('li');
+      li[el.index].setAttribute('id', el.index);
+      li[el.index].contentEditable = true;
+
+      // Create Input Element
+      input[el.index] = document.createElement('input');
+      input[el.index].setAttribute('type', 'checkbox');
+      input[el.index].classList.add('checkbox');
+      input[el.index].setAttribute('id', el.index);
```

Comment:
- [ ] [OPTIONAL] your task `index` key starts from 0 instead of one. Normally when you add the first task, it should have the `index` key equal to 1, not zero. Please correct that 🏃.